src/action.js

```diff
@@ -0,0 +1,182 @@
+import * as os from "os";
+
+import * as github from "@actions/github";
+import * as toolCache from "@actions/tool-cache";
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+
+// Define consts
+
+export const gruntworkOrg = "gruntwork-io";
+export const patcherRepo = "patcher-cli";
+export const patcherVersion = "v0.4.3";
+
+export const reportCommand = "report";
+export const updateCommand = "update";
+
+export const nonInteractiveFlag = "--non-interactive"
+export const noColorFlag = "--no-color"
+export const skipContainerFlag = "--skip-container-runtime"
+
+function osPlatform() {
+    switch (os.platform()) {
+        case "linux":
+            return "linux";
+        case "darwin":
+            return "darwin";
+        default:
+            core.setFailed("Unsupported operating system - the Patcher action is only released for Darwin and Linux");
+            return;
+    }
+}
+
+export async function openPullRequest(output, dependency, ghToken) {
```

Comment:
`output` in this case is referring to the output from `patcher` but that's a little confusing in this method. It might make more sense for it to be called something like `patcherSummary` or even `patcherRawOutput` or something like that?

Reply:
Yes - fixed!